### PR TITLE
fix bug when logging failed e2e stories

### PR DESCRIPTION
**Pull request by yobekiko**
**Proposed changes**:
- Fix error when logging failed end-2-end stories during e2e evaluation

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for raising this issue, we'll give it a review as soon as possible

File: rasa_core/evaluate.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ class EndToEndUserUtterance(UserUttered):
     Mostly used to print the full end-to-end user message in the
     `failed_stories.md` output file."""
 
-    def as_story_string(self):
+    def as_story_string(self, e2e=True):
```
Comment (ricwo) [inline rasa_core/evaluate.py]: Is this kwarg necessary?

Comment (ricwo) [reply rasa_core/evaluate.py]: I believe functionally it isn't but it preserves the signature of the parent method

Comment (yobekiko) [reply rasa_core/evaluate.py]: Yes, exactly.

**Pull request merged**